app/services/word_service.rb

```diff
@@ -1,16 +1,20 @@
+require './lib/modules/serviceable'
 class WordService
   def self.conn
    response =  Faraday.new("https://shielded-headland-00998.herokuapp.com")
   end
 
   def self.word_search
     response = conn.get("/api/v1/words?amount=")
-    JSON.parse(response.body, symbolize_names: true) 
+    # JSON.parse(response.body, symbolize_names: true) 
```

Comment:
this line as well as the pry that follows can be cleaned up

Reply:
Duly noted, updated